recipes/r-bactdating/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set version = '1.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-bactdating
+  version: {{ version|replace("-", "_") }}
+
+source:
+
+  url: https://github.com/xavierdidelot/BactDating/archive/v1.0.tar.gz
+  sha256: add9c391709e4d340d173cfb29d30ec6815d5ce89ff8038cea981260be01b151
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
```

Comment:
```suggestion
```
Please remove all comments except the cran meta data at the end of the meta.yaml.